web/src/main/java/se/inera/intyg/intygstjanst/web/service/CitizenCertificateDTOConverter.java

```diff
@@ -0,0 +1,9 @@
+package se.inera.intyg.intygstjanst.web.service;
+
+import se.inera.intyg.common.support.modules.registry.ModuleNotFoundException;
+import se.inera.intyg.intygstjanst.web.service.dto.citizen.CitizenCertificateDTO;
+import se.inera.intyg.intygstjanst.web.service.repo.model.CitizenCertificate;
+
+public interface CitizenCertificateDTOConverter {
+    CitizenCertificateDTO get(CitizenCertificate certificate, String typeName, String summaryLabel) throws ModuleNotFoundException;
```

Comment:
Call it convert(...) instead of get(...).

Remove throws ModuleNotFoundException.